doc/api/events.md

```diff
@@ -694,6 +694,37 @@ async function run() {
 run();
 ```
 
+## events.on(emitter, event)
+<!-- YAML
+added: REPLACEME
+-->
+* `emitter` {EventEmitter}
+* `eventName` {string|symbol} The name of the event being listened for
+* Returns: {AsyncIterator} that iterates `eventName` events emitted by the `emitter`
```

Comment:
Maybe it is worth to state explicitly that `value` part is an Array of event arguments (currently it is implied only by the example).